package.json

```diff
@@ -31,34 +31,32 @@
   "author": "algoan",
   "license": "ISC",
   "devDependencies": {
-    "@algoan/eslint-config": "^1.1.7",
-    "@amanda-mitchell/semantic-release-npm-multiple": "^3.8.0",
+    "@algoan/eslint-config": "^2.0.1",
+    "@amanda-mitchell/semantic-release-npm-multiple": "github:g-ongenae/semantic-release-npm-multiple#bee5dd95f9da060bd05e181402c18738204c435e",
```

Comment:
I am not a fan of this syntax and IMO it would be safer to use algoan's fork.

Reply:
Fixed in d3f66d7

Reply:
thank you